src/pages/Staking/index.tsx

```diff
@@ -208,6 +206,20 @@ const Staking = (): JSX.Element => {
   );
   useErrorHandler(stakedAmountAndEndBlockError);
 
+  const {
+    isIdle: transactionFeeReserveIdle,
+    isLoading: transactionFeeReserveLoading,
+    data: transactionFeeReserve,
+    error: transactionFeeReserveError
+  } = useQuery<GovernanceTokenMonetaryAmount, Error>(
+    ['transactionFeeReserve', address],
```

Comment:
The key is hardcoded. I think we can define it in the underlying fetcher as well. You can take a look at other fetchers.